.github/workflows/scanner-build.yaml

```diff
@@ -14,13 +14,50 @@ on:
     - synchronize
 
 jobs:
+  define-job-matrix:
+    outputs:
+      matrix: ${{ steps.define-job-matrix.outputs.matrix }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Define the matrix for build jobs
+      id: define-job-matrix
+      run: |
+        source './scripts/ci/lib.sh'
+
+        matrix='{ "build_and_push": { "name":[], "goos":[], "goarch":[] } }'
+
+        # The base matrix
+        matrix="$(jq '.build_and_push.name += ["default"]' <<< "$matrix")"
+        matrix="$(jq '.build_and_push.goos += ["linux"]' <<< "$matrix")"
+        # If this is updated, be sure to update architectures in push-manifests job below.
+        matrix="$(jq '.build_and_push.goarch += ["amd64", "arm64", "ppc64le", "s390x"]' <<< "$matrix")"
+
+        # Conditionally add a prerelease build (binaries built with GOTAGS=release)
+        if ! is_tagged; then
+          if ! is_in_PR_context || pr_has_label ci-build-prerelease; then
+            matrix="$(jq '.build_and_push.name += ["prerelease"]' <<< "$matrix")"
```

Comment:
Not seeing any checks in the PR output showing `prerelease`, assuming because the PR label doesn't exist.  The logic 'seems OK' but difficult to review without seeing it run successfully.  

Any concerns adding the label to this PR (or whatever else is needed) to see if the workflow generates the `prerelease` builds?

Reply:
It exists, I just didn't add it. Added it now. Let's see what happens :)

Reply:
Looks like it was built with `-tags release`